neo/bin/api_server.py

```diff
@@ -181,6 +189,20 @@ def main():
         else:
             print("Logging to stdout and stderr")
 
+    if args.wallet:
+        if not os.path.exists(args.wallet):
+            print("Wallet file not found")
+            return
+        passwd = prompt("[password]> ", is_password=True)
```

Comment:
We could allow an environment variable for the password, so this could run more easily supervised (eg. systemd). For instance `NEO_PYTHON_JSONRPC_WALLET_PASSWORD`, and if set it would skip the manual prompt. Just an idea, what do you think?

Reply:
Yes definitely. :+1: